### DISCUSSION TITLE
High Precision Mouse drops input bug

### DISCUSSION THREAD

**Discussion by CpDowns** [ppy/osu#27092] [Q&A]
### Type

Game behaviour

### Bug description

Even though it might be wierd but I use a tablet from Wacom (a fairly old one) and back in the day in stable I found it best for me to use relative mode and emulate my pen as a mouse in the Wacom driver. I then used raw input to adjust the speed in Osu (stable) and this worked without issue even to this day. 

Now that lazer is getting a lot of updates I decided to revisit it using the same settings as in stable, but if I try to use high precision mouse (hpm) in lazer with my tablet set to emulate a mouse in wacom the cursor starts to drop inputs consistently every 1-2 seconds making it unplayable. Without hpm it works fine but the sensitivity is wrong ofc.

Its a pretty wierd bug so I understand if this doesnt get fixxed right away but i would like to continue to use my tablet as i do in stable and not have to readapt for lazer.

The video attached shows all settings in wacom aswell as in osu(stable) and lazer and the impact that hpm has on the controlls in both versions.

Kind Regards, CpDowns

### Screenshots or videos

https://youtu.be/sOBAq_fNU5w (video explaining the bug)

### Version

2024.131.0-lazer

### Logs

[compressed-logs.zip](https://github.com/ppy/osu/files/14202110/compressed-logs.zip)

**Comment by Susko3** [comment]
Could you open global statistics and scroll down to the `Input - ` sections and record a video of the bug in action.
